packages/components/data-table/package.json

```diff
@@ -22,6 +22,7 @@
     "@babel/runtime": "^7.20.13",
     "@babel/runtime-corejs3": "^7.20.13",
     "@commercetools-uikit/accessible-button": "19.2.0",
+    "@commercetools-uikit/data-table-manager": "19.2.0",
```

Comment:
I'm concerned about this. Now whoever uses the `DataTable` component, will include the `DataTableManager` in their application JS bundles even if they don't use the latter component.

An idea would be to export the provider from the `DataTableManager` package as an independent entry point (we do something similar in the icons package) so consumers can import only the provider code.

Reply:
On it. Thank you

Reply:
updated here: https://github.com/commercetools/ui-kit/pull/2807/commits/e02436f523fc54b93c131ade1cb3b5b26bb4a20a